video/netsource.h

```diff
@@ -32,12 +32,12 @@ class NetSource : public Input
     NetSource(Net::Node &node)
     {
         // handle incoming IMG commands
-        node.addCommandHandler("IMG", [this](auto &node, auto &command) {
+        node.addCommandHandler("IMG", [this](Net::Node &node, const Net::Command &command) {
             onCommandReceived(node, command);
```

Comment:
It's weird that this doesn't work on your compiler, because this is definitely part of the C++14 standard.

Obvs we don't *need* this here but maybe we're building something with C++11 by mistake?